spec.html

```diff
@@ -28085,9 +28396,13 @@ <h1>Static Semantics: ImportEntries ( ): a List of ImportEntry Records</h1>
         <emu-alg>
           1. Return a new empty List.
         </emu-alg>
-        <emu-grammar>ImportDeclaration : `import` ImportClause FromClause `;`</emu-grammar>
+        <emu-grammar>
+          ImportDeclaration :
+            `import` ImportClause FromClause `;`
+            `import` ImportClause FromClause WithClause `;`
```

Comment:
```suggestion
            `import` ImportClause FromClause WithClause? `;`
```
(If they're combined via `?` in the definition, we prefer to combine them via `?` in the reference.)